src/plugins/AriaPopup.js

```diff
@@ -62,15 +64,18 @@ export default class AriaPopup extends Aria {
    * Add initial attributes, establish relationships, and listen for events
    */
   ariaSetup() {
+    this.loadOpen = this._loadOpen;
+    this.isExpanded = this._loadOpen;
```

Comment:
you might consider using a [getter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get) for one or both of these properties, not a requirement though.

Reply:
I gave this a shot and it seems like extra code to accomplish the exact same thing. The only time we want to access `_loadOpen` is in `ariaSetup`, otherwise `ariaReset` (which calls `ariaSetup`) breaks.